components/raftstore/src/store/fsm/apply.rs

```diff
@@ -2821,28 +2824,37 @@ where
     ) -> Result<(AdminResponse, ApplyResult<EK::Snapshot>)> {
         let region_id = self.region_id();
         let region_state_key = keys::region_state_key(region_id);
+        let is_in_flashback = req.get_cmd_type() == AdminCmdType::PrepareFlashback;
+        // We need to make sure the `RegionLocalState` read from the engine is latest,
+        // so the flush is needed before this function being called.
         let mut old_state = match ctx
             .engine
             .get_msg_cf::<RegionLocalState>(CF_RAFT, &region_state_key)
         {
             Ok(Some(s)) => s,
             _ => {
-                return Err(box_err!("failed to get region state of {}", region_id));
+                return Err(box_err!(
+                    "failed to get region local state of {}",
+                    region_id
+                ));
             }
         };
-        let is_in_flashback = req.get_cmd_type() == AdminCmdType::PrepareFlashback;
```

Comment:
Why need to move this line?

Reply:
Just some tiny reorganization to make the code structure clear.